docs/main_text.md

```diff
@@ -534,15 +534,24 @@ Below is the detailed folder structure:
 
 You can use the refine.bio API to build your own applications utilizing the refine.bio processed data.
 You can also select samples for aggregation and download via the API with additional options for download (e.g., without quantile normalization, selecting sample-specific `quant.sf` files).
-Our quantile normalization targets and transcriptome indices are available only via the API (see [_Quantile normalizing your own data with refine.bio reference distribution_](#quantile-normalizing-your-own-data-with-refine.bio-reference-distribution) and [_Transcriptome index_](#transcriptome-index), respectively).
+Our quantile normalization targets and transcriptome indices are available only via the API (see [_Quantile normalizing your own data with refine.bio reference distribution_](#quantile-normalizing-your-own-data-with-refine-bio-reference-distribution) and [_Transcriptome index_](#transcriptome-index), respectively).
 
 **For more information see our API documentation at <a href = "https://api.refine.bio/" target = "blank">api.refine.bio</a>.**
 
-# Use Cases for Downstream Analysis
+# Downstream Analysis with refine.bio Examples
 
-Our <a href = "https://github.com/AlexsLemonade/refinebio-examples/tree/v1.0" target = "blank">`refinebio-examples`</a> repo includes a number of different analyses you can perform with data from refine.bio. We include examples in the R programming language, and where applicable, <a href = "http://genepattern-notebook.org/example-notebooks/" target = "blank">GenePattern Notebooks</a> and scripts to prepare refine.bio data for use with GenePattern. The following examples are included:
+Our <a href = "https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html" target = "blank">refine.bio examples site</a> includes a number of different analyses you can perform with data from refine.bio in the R programming language. 
```

Comment:
My guess is that when we get a "main" page we will switch this link to that?

Reply:
Yes, that's what I'm thinking.